14/umbraco-cms/fundamentals/setup/install/running-umbraco-on-docker-locally.md

```diff
@@ -0,0 +1,97 @@
+﻿---
+description: "Running Umbraco on docker locally using docker compose"
+---
+
+# Running Umbraco in Docker using Docker Compose
+
+To aid in developing Umbraco with additional services, the templates can provide the requisite files to run Umbraco with a SQL Server database in Docker.
+This setup is designed to be used for local development, and not for production.
+
+## Prerequisites
+
+Before you can run Umbraco in Docker, you need to have the following installed:
+* Version 14.3.0 or higher of the Umbraco Templates
+* Docker Desktop
+
+## Installing
+
+Installing Umbraco with the Docker file and Docker Compose file is a two-step process.
+
+First, create a folder to hold your project and enter that folder.
+
+```bash
+mkdir MyDockerProject
+cd MyDockerProject
+```
+Next create your Umbraco project using the Umbraco Templates, and remember to use the `--add-docker` flag to include the Docker files.
+Conventionally this is named the same as the folder, but is not a requirement.
+
+```bash
+dotnet new umbraco -n MyDockerProject --add-docker
+```
+
+Now we need to add some additional files to make docker compose work. We can do this using the `umbraco-compose` template, passing the project name we specified earlier to the `-P` parameter:
```

Comment:
Can we use steps here? 😇 

```suggestion
1. Create a folder to hold your project and enter that folder.

```bash
mkdir MyDockerProject
cd MyDockerProject
```

2. Create your Umbraco project using the Umbraco Templates, and remember to use the `--add-docker` flag to include the Docker files.

Conventionally this is named the same as the folder, but it is not a requirement.

```bash
dotnet new umbraco -n MyDockerProject --add-docker
```

Now we need to add some additional files to make docker compose work. We can do this using the `umbraco-compose` template, passing the project name we specified earlier to the `-P` parameter:
```